_maps/shuttles/whiteship_birdshot.dmm

```diff
@@ -146,7 +146,7 @@
 	pixel_y = 3
 	},
 /obj/item/reagent_containers/blood,
-/obj/item/surgery_tray/full{
+/obj/effect/spawner/surgery_tray/full{
```

Comment:
Is it really necessary for this to be the case on every whiteship and shuttle, as well as on the station? 
It's fine for some places to actually map in the item.

Reply:
it's not necessary nor unnecessary. it just is